src/midi/midi.c

```diff
@@ -238,3 +238,117 @@ static inline void initiate_transfer()
     state = STATE_TRANSFER;
     midi_transfer_progress = 0;
 }
+
+const struct midi_command midi_channels_cc[][15] = {
```

Comment:
This information is constant, so it should be read out from the flash directly instead of using RAM to store it. You tell the compiler to do this by adding PROGMEM to the end of the declaration. When reading values you then use some special functions to read from the flash/program memory.
You can see an example of how this is done in parameter/parameter.c.